CHANGELOG.md

```diff
@@ -53,6 +53,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * The `-R`/`--repository` path must be a valid workspace directory. Its
   ancestor directories are no longer searched.
 
+* When `jj init` is used to create a colocated repository, the commits that are
```

Comment:
Nit: The bug is not specific to colocated repos. The reproduction I shared in the bug doesn't use a colocated repo, for example. So something like this might work:

```
Fixed a crash when trying to access a commit that's never been imported into the jj repo from a Git repo. They will now be considered as non-existent if referenced explicitly instead of crashing.
```

Reply:
Done.